src/Utilities/Compiler/Options/AnalyzerOptionsExtensions.cs

```diff
@@ -205,31 +205,70 @@ static SymbolNamesOption.NameParts GetParts(string name)
             }
         }
 
+        public static SymbolNamesOption GetInheritanceExcludedSymbolNamesOption(
```

Comment:
I suggest changing this method to take the default forced value as a parameter, and let the caller provide this value.

Reply:
Ok so we don't always for the `N:System.*` pattern, do we?

Reply:
Nevermind this was a stupid comment.